website/docs/foundations/breakpoints/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+Different defined widths that help standardize responsive experiences across the HashiCorp product suite.
```

Comment:
I'm seeing this sentence appear at the top of the Guidelines. It doesn't seem needed though. (Seems redundant to the other nearby text)

<img width="756" alt="image" src="https://github.com/user-attachments/assets/3e904409-28bf-4dde-bc88-4c7e06d82f9a" />